### DISCUSSION TITLE
Make install Error

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [GoogleCloudPlatform/cluster-toolkit#803] [Q&A] [answered]
when I am doing make install, this error is showing, please help me to resolve this error

/bin/bash: line 1: ../../../config/make_manpage.pl: Permission denied
make[2]: *** [Makefile:2215: opal_crs.7] Error 126
make[2]: Leaving directory '/home/nikhil/openmpi-4.1.1/opal/mca/crs'
make[1]: *** [Makefile:2383: install-recursive] Error 1
make[1]: Leaving directory '/home/nikhil/openmpi-4.1.1/opal'
make: *** [Makefile:1901: install-recursive] Error 1

**Comment by cboneti** [comment, marked as answer]
Hi, 

I have not seen this error before.  Looking at your history, it seems you are using openmpi-4.1.1. Where are you running `make install`? is it in the hpc-toolkit folder?

**Reply by cboneti** [reply]
since we had no answer here, I am assuming this was not a problem in the HPC Toolkit and will mark this as answered.  Please let me know if otherwise.
